app/commands/github/pull_request/create_or_update.rb

```diff
@@ -7,25 +7,12 @@ class CreateOrUpdate
 
       def call
         pull_request = ::Github::PullRequest.create_or_find_by!(node_id:) do |pr|
-          pr.number = attributes[:number]
-          pr.title = attributes[:title]
-          pr.repo = attributes[:repo]
-          pr.author_username = attributes[:author_username]
-          pr.merged_by_username = attributes[:merged_by_username]
-          pr.data = attributes[:data]
+          pr.attributes = attributes.except(:reviews)
```

Comment:
What's `attributes` here? If this comes from the webhook, then this feels quite risky to me, as it could override our internal db columns (like id, or created_at). I think we'd be better having an accept-list of keys here. If it's pre-sanitized before this point, you can likely ignore this comment though.

Reply:
There is sanitization happening beforehand. The command is called in two locations:

- https://github.com/exercism/website/blob/main/app/jobs/process_pull_request_update_job.rb#L8-L17
- https://github.com/exercism/website/blob/51fad058a10212ab8dbfcf2210a47b3f3259a6bf/app/commands/github/pull_request/sync_repo.rb#L10-L19